### ISSUE TITLE
Editing code causes "jumps" in the code editors scroll position

### ISSUE THREAD

**Issue by maltenuhn** [concrete-utopia/utopia#1025]
**Describe the bug**
When making edits (eg deletion or pasting) code, the code editor jumps visually. It shouldn't. 

**To Reproduce**
Steps to reproduce the behavior. For canvas tasks, take care to note down all relevant key holds, keys typed, selected state etc. E.g:

1. Paste the below code into a new project, into `storyboard.js`
2. Copy and paste the large div with multiple rows of style info below itself

Observed: file scrolls to the top, then back

Result: Mouse pointer is still indicating resize, even though A can't be resized where the mouse is, an B is no longer selected or resizeable when dragging the mouse where it is.

**Expected behaviour**
No scrolling at all should occur except the built-in VSCode one.

Likely related to #1024 

**Sample Code**

```
/** @jsx jsx */
import * as React from 'react'
import { Scene, Storyboard, jsx } from 'utopia-api'
export var App = (props) => {
  return (
    <div
      style={{
        padding: 20,
        width: '100%',
        height: '100%',
        backgroundColor: '#FFFFFF',
        position: 'relative',
      }}
    >
      <div
        style={{
          width: 270,
          height: 215,
          background: 'hsl(0,0%,97%)',
          boxShadow: 'rgba(0, 0, 0, 0.1) 0px 0px 0px 1px, rgba(0, 0, 0, 0.1) 0px 4px 11px',
          backgroundColor: 'rgb(247, 247, 247, 1)',
        }}
      >
        <div
          style={{
            paddingLeft: 8,
            paddingRight: 8,
            fontFamily: 'Inter',
            fontSize: 11,
            color: 'hsl(0,0%,10%)',
            display: 'flex',
            alignItems: 'center',
            height: 34,
          }}
        >
          <span style={{ fontWeight: 600 }}>Item Selector</span>
          
        </div>
        
      </div>
    </div>
  )
}
export var storyboard = (
  <Storyboard>
    <Scene
      component={App}
      props={{}}
      style={{ position: 'absolute', left: 0, top: 0, width: 375, height: 812 }}
    />
  </Storyboard>
)

```

**Comment by Rheeseyb** [comment]
I'm pretty sure this is related to #1024 (as mentioned in there)

**Comment by Rheeseyb** [comment]
Closed by #1100
